### DISCUSSION TITLE
Stuck notification

### DISCUSSION THREAD

**Discussion by TriMoon** [community/community#43032] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have a notification i can't get rid of, even after multiple reloads etc...
The notification in question is not even shown, but it is listed in the listings as can be seen in the screenshot.
How can i get rid of this one?

![image](https://user-images.githubusercontent.com/1723378/210164229-b59324b2-3808-4348-a095-bc472f2d10cd.png)

Similar to #32380,#32712

**Comment by radar-blip** [comment]
There are some suggestions here that should help you: #6874

Like this:

    Goto Done.
    Select all
    Move to inbox
    Goto Inbox
    Select all
    Mark as read
    Mark as Done

**Reply by TriMoon** [reply]
I had already tried all ways using the GUI, but none worked...
_And i'm not willing to fiddle using the API via command-line..._

**Reply by stringang** [reply]
The suggested method is useful, my problem is solved.

**Reply by ryanmkurtz** [reply]
This worked for me too.  I had to select more than the current screen showed but after I selected enough of my history and performed the steps, it worked.

**Reply by runoncedev** [reply]
First time it didn't seem to work. Tried once more buy this time moving a lot more of done notifications to the inbox and it worked!

**Reply by LiberiFatali** [reply]
This works. Be aware that if you have multi-pages, after selecting the first page it will show a link to select the rest.

**Reply by vgdh** [reply]
That method magically works.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @TriMoon ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by TriMoon** [reply]
Thanks, but thats all i could do using the presented UI while creting this topic...
_We're not able to manually select categories..._

**Reply by queenofcorgis** [reply]
Hi there - glad to see your notification is no longer stuck, but would you mind elaborating on what you mean by you're not manually able to select categories? We want to make sure you have the best experience in Discussions.

**Reply by TriMoon** [reply]
Well TBH, i forgot the exact steps i took to post this issue because it was my first time on this board.
But if i would have been able to select categories im sure i would have done so, because im used to use categories all over the web :wink: 
Anyhow the bug has dissolved itself as i mentioned in my marked post :vulcan_salute:

**Reply by queenofcorgis** [reply]
Thanks for the quick response - if you encounter anything like that with the categories again - please give me a ping!

**Comment by TriMoon** [comment, marked as answer]
Somehow they seem to have fixed my issue _**Behind the scenes**_ because it's gone without me doing anything except posting this thread :rofl:

**Comment by ryanmkurtz** [comment]
I too have a stuck notification now.

**Reply by fyrak1s** [reply]
https://github.com/orgs/community/discussions/43032#discussioncomment-9456373

**Comment by runoncedev** [comment]
I'm having the same problem I think.

It seems to be coming from this particular repository but I tried all kind of filters and nothing shows up:

![image](https://user-images.githubusercontent.com/1458916/216998991-00198aef-11b4-4cbe-a9dc-c1c57e8136e3.png)

It's getting pretty annoying, anyway I can get rid of it?

**Reply by fernandoxu** [reply]
<img width="1657" alt="image" src="https://user-images.githubusercontent.com/20871468/219956125-77e4f878-b829-4dd4-901d-06467d9eac3e.png">
<img width="1308" alt="image" src="https://user-images.githubusercontent.com/20871468/219956261-827b752b-7385-46ce-8418-40507ffe4a48.png">
same! even if I set stop ignoring

**Comment by fvlasie** [comment]
I am experiencing the same issue. I tired all the steps mentioned here but I cannot resolve it. I have 3 repos with notifications that do not show when I click on them so i cannot mark them as done.

**Reply by fyrak1s** [reply]
try https://github.com/orgs/community/discussions/43032#discussioncomment-9456373

**Comment by M-Reimer** [comment]
I also have a "stuck notification". Still someone watching here who can fix this for me? I tried everything...

**Reply by fyrak1s** [reply]
Same here. No notifications left but still getting the blue dot.

**Reply by M-Reimer** [reply]
Yeah but noone seems to care. Still have the problem.

**Reply by fyrak1s** [reply]
This fixed it for me: https://github.com/orgs/community/discussions/43032#discussioncomment-9456373

**Comment by andrewjswan** [comment]
I also have a "stuck notification". Still someone watching here who can fix this for me? I tried everything...

**Reply by fyrak1s** [reply]
This fixed it for me: https://github.com/orgs/community/discussions/43032#discussioncomment-9456373

**Comment by M507** [comment]
Same issue here..

**Reply by fyrak1s** [reply]
This fixed it for me: https://github.com/orgs/community/discussions/43032#discussioncomment-9456373

**Comment by fyrak1s** [comment]
[This](https://github.com/orgs/community/discussions/6874#discussioncomment-9387397) has resolved it for me.

You have to [create a GitHub token](https://github.com/settings/tokens/new), select "Access notifications" and create it.

Copy the token and in your console run `export TOKEN=<your-token>`.

Then copy and paste this into your console and run it:
```bash
curl -X PUT \
   -H "Accept: application/vnd.github.v3+json" \
   -H "Authorization: token $TOKEN" \
    https://api.github.com/notifications \
   -d @<(cat <<EOF
   {
     "last_read_at": "$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
   }
EOF
)
```

**Reply by thefrana** [reply]
This fixed my issue even now, thanks! It was driving me crazy.

**Comment by SonoSooS** [comment]
(see [this](https://github.com/orgs/community/discussions/111305#discussioncomment-9639200) for full explaination)

You can go to [https://github.com/notifications](https://github.com/notifications), and run this script in Inspect Element console:
```Javascript
(function(doc,endpoint)
{
    const a=doc.querySelector('form[action="'+endpoint+'"]');
    const b=doc.createElement("input");
    b.type='hidden';
    b.name='query';
    b.value="";
    a.appendChild(b);
    const c=doc.createElement("input");
    c.type='hidden';
    c.name='mark_all';
    c.value="1";
    a.appendChild(c);
    a.submit();
})(document,"/notifications/beta/mark");
```
If it doesn't make the ghost notification read, replace `mark` with `unmark` in the last line of the script, run that first, and then run `mark` again, and that should make the ghost notification read.  
Sadly this doesn't get rid of the ghost notification (so notification pagination will be still broken), but at least the unread bubble can finally go extinguish :)  
This method also works if you can't dig up more notifications to make it at least 26 to use the scriptless method.

**Reply by roniemartinez** [reply]
Worked for me but now the repo is still stuck on the Repo list.
<img width="300" src="https://github.com/community/community/assets/2573537/511d87f0-a767-44b7-83b3-586647dcfcc7">

**Reply by SonoSooS** [reply]
It should time out after ~30d, I think.

Since we have no way to query *all* notifications, and thus we can't flag it as done, you'll have to live with it until it either magically pops back up, or until it times out by itself.

**Reply by arose26** [reply]
Worked for me when all else failed (the cause was a notif in a repo that was deleted). Thanks!

**Reply by akash1520** [reply]
Thank you, it worked now I feel relieved

**Reply by HansBrende** [reply]
This worked for me!! Thank you so much.

**Comment by qndel** [comment]
Got a stuck notification because dude opened a discussion in my repo, was getting notifications from it, then dude somehow got DELETED along with his discussion yet notification is permastuck, this is so annoying

**Comment by HansBrende** [comment]
I'm having the same issue from a repository that appears to be deleted... I have no way to mark anything as "done" because this repository literally does not exist. How do I make it stop 😭

<img width="87" height="76" alt="Screenshot 2025-09-27 at 2 29 31 PM" src="https://github.com/user-attachments/assets/2f6903b6-56cd-454f-a814-6fc1a01b39fd" />
<img width="301" height="88" alt="Screenshot 2025-09-27 at 2 29 14 PM" src="https://github.com/user-attachments/assets/3533a672-2269-4887-a82d-4108baa50d2e" />

<img width="561" height="288" alt="Screenshot 2025-09-27 at 2 35 59 PM" src="https://github.com/user-attachments/assets/a9f31f0e-0b28-4a48-b958-287633200e62" />

**Reply by BenceSzalai** [reply]
Same here. Also I cannot recall ever interacting with that repo. I guess they found a way to use the repo for spam, github deleted the repo and now we are stuck with an invisible permanently unread notification from a non-existent repo. Fun.

**Reply by cmpute** [reply]
Same here. It's weird that delete a repo will not delete the notification along with.

**Comment by deathaxe** [comment]
I also suffered from a zombi notification caused by that removed organization's repository, thus suggested manual move to inbox and dismiss workflow failed.

I was finally able to resolve the issue with following steps:

1. Install the [GitHub CLI](https://cli.github.com/).
2. Authenticate from GitHub CLI.
3. Use `gh api notifications` to get a JSON containing all your notifications.
4. Note the notification (thread) ID (`<id>`) of the bad repository.
5. Execute `gh api --method PATCH notifications/threads/<id>` to read all the notifications of the given thread ID.

Note: gh might ask to upgrade your authentication to be able to modify notifications.

**Reply by HansBrende** [reply]
Ok so first I tried the post 3 posts above yours which required less work (just copy-and-paste into console), and it actually worked! Glad both of our phantom notifications have gone away.

**Reply by BenceSzalai** [reply]
Thanks, this worked!

**Reply by cmpute** [reply]
Thanks so much! This works like a charm
